packages/blockchain-utils-validation/src/index.ts

```diff
@@ -23,9 +23,17 @@ const findDID = (did: string): string | undefined => did.match(/(did:\S+:\S+)/)?
 export async function validateLink(proof: LinkProof): Promise<LinkProof | null> {
   // version < 2 are always eip155 namespace
   let namespace = ethereum.namespace
+
+  // Handle legacy CAIP links
+  if (proof.account.includes('@')) {
+    const [address, chainId] = proof.account.split('@')
+    proof.account = new AccountId({ address, chainId }).toString()
+  }
```

Comment:
This code path is not tested afaict

Reply:
Yeah - looks like theres no tests which use this at all